doc/specs/#658 - WinGet Download.md

```diff
@@ -0,0 +1,209 @@
+---
+author: Roy MacLachlan RDMaclachlan/roy.maclachlan@microsoft.com
+created on: 2023-02-09
+last updated: 2023-02-09
+issue id: 658
+---
+
+# Spec Title
+
+"For [#658](https://github.com/microsoft/winget-cli/issues/658)"
+
+## Abstract
+
+This spec describes the functionality and high-level implementation design for the downloading of applications using the Windows Package Manager.
+
+## Inspiration
+
+This is inspired by customer feedback, and a need for broader application deployments:
+* Customers want to share the installer with an offline device. 
```

Comment:
I'd think for this to be worth it, we'd also have to have a way to tell the install command to use already downloaded installers. Probably something like `--local-installer-location` that defaults to the `userDefaultAppOutput` directory.

Reply:
I'm not sure I understand? Why wouldn't they use the local manifest for local installations?